controls/SV-221584.rb

```diff
@@ -21,4 +21,12 @@
   tag legacy: ["SV-57639","V-44805"]
   tag cci: ["CCI-002605"]
   tag nist: ["SI-2 c"]
+
+  domain_role = command('(Get-Item (Get-ItemProperty \'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe\').\'(Default)\').VersionInfo.ProductVersion')
```

Comment:
Consider using inSpec's powershell resource to execute powershell commands and compare against commands' outputs. Example:

  current_chrome_version_command = <<-EOH
  (Get-Item (Get-ItemProperty 'HKLM:\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\chrome.exe').'(Default)').VersionInfo.ProductVersion    
  EOH

  describe powershell(current_chrome_version_command) do
    its('stdout') { should cmp >= '74.0.0' }
  end

Reply:
Its giving me a **Invalid escape character syntax Error**

Reply:
https://stackoverflow.com/questions/52457766/how-can-i-get-google-chromes-version-number 

I think the issue is the \\

Reply:
The double slashes have to be there, unless you'll get the syntax error

Reply:
Can we use the simplier command of `$(Get-Package -Name "Google Chrome").Version` in powershell?